qubesmanager/backup.py

```diff
@@ -225,11 +225,11 @@ def shutdown_all_running_selected(self):
             self.blk_manager.check_if_serves_as_backend(vm)
 
         reply = QMessageBox.question(None, self.tr("VM Shutdown Confirmation"),
-             unicode(self.tr(
```

Comment:
I'm pretty sure some of those dropped `unicode().format()` calls are reachable, please test.